app/uk/gov/hmrc/cdsreimbursementclaimfrontend/views/supportingevidence/check_your_answers.scala.html

```diff
@@ -15,48 +15,102 @@
 
     @key = @{"supporting-evidence.check-your-answers"}
 
-    @title = @{messages(s"$key.title")}
-
-    @layout(pageTitle = Some(s"$title"), signedInUserDetails = request.signedInUserDetails, suppressBackLink = true) {
+    @title = @{messages(s"$key.title", evidences.size, if (evidences.size > 1) "s" else "")}
 
-        @pageHeading(title)
-
-        @paragraph(Html(messages(s"$key.help-text")), Some("govuk-inset-text govuk-!-margin-bottom-8"))
+    @postAction = @{ routes.SupportingEvidenceController.checkYourAnswersSubmit(journey) }
 
-        @summary(SummaryList(fileUploadCya.makeUploadedFilesRows(journey, evidences)))
+    @hasErrors = @{form.hasErrors || form.hasGlobalErrors}
 
-        @if(evidences.size < maxUploads){
-            @paragraph(Html(messages(s"$key.add-another-file", routes.SupportingEvidenceController.uploadSupportingEvidence(journey).url)), Some("govuk-body"))
-        }
+    @layout(pageTitle = Some(s"$title"), signedInUserDetails = request.signedInUserDetails, suppressBackLink = true, hasErrors = hasErrors) {
 
-        @formWithCSRF(routes.SupportingEvidenceController.checkYourAnswersSubmit(journey), 'novalidate -> "novalidate") {
+        @errorSummary(form.errors)
 
-            @submitButton("button.confirm-and-continue")
+        @pageHeading(title)
 
+        @paragraph(Html(messages(s"$key.help-text")), Some("govuk-body govuk-!-margin-bottom-6"))
+
+        <dl class="govuk-summary-list govuk-!-margin-bottom-8">
+            <div class="govuk-summary-list__row">
+                <dt class="govuk-summary-list__key"></dt>
+                <dd class="govuk-summary-list__actions"></dd>
+            </div>
+            @evidences.toList.map { evidence =>
+                <div class="govuk-summary-list__row">
+                    <dt class="govuk-summary-list__key">
+                        @defining(evidence.fileName) { name =>
+                            @name<br />
+                            <span style="font-weight: normal">
+                                @{evidence.documentType.fold("")(documentType =>
+                                    messages(s"supporting-evidence.choose-document-type.document-type.d${documentType.index}")
+                                )}
+                            </span>
+                        }
+                    </dt>
+                    <dd class="govuk-summary-list__actions">
+                        @defining(routes.SupportingEvidenceController.deleteSupportingEvidence(journey, evidence.uploadReference, addNew = false).url) { changeUrl =>
+                            <a class="govuk-link" href=@changeUrl>
+                                @{messages(s"cya.remove")}
+                            </a>
+                        }
+                    </dd>
+                </div>
+            }
+        </dl>
```

Comment:
@oddbunchmedia @mikail-khan Hey guys please be aware of this ^^. Perhaps might want to extract to some components later on....